diff --git a/node_modules/truffle-typings/index.d.ts b/node_modules/truffle-typings/index.d.ts
index eae5e74..6bf85b2 100644
--- a/node_modules/truffle-typings/index.d.ts
+++ b/node_modules/truffle-typings/index.d.ts
@@ -5,13 +5,10 @@
 /// <reference types="mocha" />
 
 declare type BN = import("bn.js");
-declare type Web3 = import("web3");
 
 declare const assert: Chai.AssertStatic;
 declare const expect: Chai.ExpectStatic;
 
-declare const web3: Web3;
-
 declare const artifacts: Truffle.Artifacts;
 
 /**
